localstack-core/localstack/testing/pytest/in_memory_localstack.py

```diff
@@ -53,10 +53,14 @@ def pytest_runtestloop(session: Session):
 
     from localstack.testing.aws.util import is_aws_cloud
 
-    if is_env_true("TEST_SKIP_LOCALSTACK_START") or is_aws_cloud():
+    if is_env_true("TEST_SKIP_LOCALSTACK_START"):
         LOG.info("TEST_SKIP_LOCALSTACK_START is set, not starting localstack")
         return
 
+    if is_aws_cloud() and not is_env_true("TEST_FORCE_LOCALSTACK_START"):
+        LOG.info("Test running against aws, not starting localstack")
+        return
```

Comment:
nit: while it's a bit more code, I'd argue it would be cleaner here as well to have a separate condition just for the `TEST_FORCE_LOCALSTACK_START` case to log *why* we're starting localstack even though we're targeting AWS

Reply:
Good idea! I will add that